scripts/ci/run_full_ci_pipeline.py

```
@@ -179,169 +179,167 @@ def run_unit_tests(self) -> bool:
+
     def test_gpu_compatibility(self) -> bool:
         """Test GPU compatibility if available."""
         logger.info("🖥️ Testing GPU compatibility...")
-        
+
         try:
             import torch
-            
+
             if not torch.cuda.is_available():
                 logger.info("ℹ️ No GPU available, skipping GPU tests")
```

Ruff (0.12.2)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)